core/trino-main/src/main/java/io/trino/testing/TestingAccessControlManager.java

```diff
@@ -125,9 +125,14 @@
     private BiPredicate<Identity, String> denyIdentityTable = IDENTITY_TABLE_TRUE;
 
     @Inject
+    public TestingAccessControlManager(TransactionManager transactionManager, EventListenerManager eventListenerManager, AccessControlConfig accessControlConfig)
+    {
+        super(transactionManager, eventListenerManager, accessControlConfig, DefaultSystemAccessControl.NAME);
+    }
+
     public TestingAccessControlManager(TransactionManager transactionManager, EventListenerManager eventListenerManager)
```

Comment:
`@VisibleForTesting`?

Reply:
These classes are in general for testing only so this annotation could be misleading IMHO